test/parallel/test_tensorflow2_keras.py

```diff
@@ -16,27 +16,24 @@
 """Tests for horovod.tensorflow.keras in TensorFlow 2."""
 
 import math
-import tensorflow as tf
-import numpy as np
 import os
 import warnings
-
 from distutils.version import LooseVersion
 
+import horovod.tensorflow.keras as hvd
+import numpy as np
 import pytest
-
+import tensorflow as tf
```

Comment:
Is there a reason of moving the order of importing packages?
I think the convention is: 1. import third party packages; 2. from .. import third party packages; 3. horovod internal stuff.

Reply:
It was auto-formatted via pylance on VSCode. I disabled it now in consideration of convention.